public/js/user.js

```diff
@@ -269,17 +269,18 @@ function saveMarker(e) {
     sites[id].editBubbleOpen = false;
     sites[id].marker.setDraggable(false);
     var json = JSON.stringify(site);
+    var isUpdateRequest = isValidMarkerId(id);
     $.ajax({
-      //If marker id is a uuid, its an update request to '/marker/[id]'
-      //else it is an add request to '/marker'
-      url: "/marker/"+(isValidMarkerId(id)?id:""),
-      type: "POST",
+      //If marker id is a uuid, its an PATCH update request to '/marker/[id]'
+      //else it is an POST create request to '/marker'
+      url: "/marker/"+(isUpdateRequest?id:""),
```

Comment:
Don't send a trailing slash when you create a marker:
```node
url: "/marker" + (isUpdateRequest ? "/"+id : ""),
```

Reply:
Done! :)